modules/eloqua_api_auth_fallback/src/Form/Settings.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Drupal\eloqua_api_auth_fallback\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Link;
+use Drupal\Core\Url;
+
+/**
+ * Eloqua API Fallback Authentication Settings.
+ */
+class Settings extends ConfigFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'eloqua_api_auth_fallback_settings';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+    return ['eloqua_api_auth_fallback.settings'];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $config = $this->config('eloqua_api_auth_fallback.settings');
+
+    $form['credentials'] = [
+      '#type' => 'fieldset',
+      '#title' => $this->t('Eloqua API resource owner password credentials'),
+    ];
+
+    $form['credentials']['help'] = [
+      '#type' => '#markup',
+      '#markup' => $this->t('For more information on authenticate using a resource owner password credentials grant, refer this @link.',
+        [
+          '@link' => Link::fromTextAndUrl('https://docs.oracle.com/cloud/latest/marketingcs_gs/OMCAB/Developers/GettingStarted/Authentication/authenticate-using-oauth.htm#resource-owner-password-credentials-grant',
```

Comment:
Can this be text like "**refer to API Documentation for resource owner password credentials grant**" with hyperlink?

Reply:
Link updated